helm/cyclone/templates/server.yaml

```diff
@@ -77,6 +77,9 @@ spec:
       - name: server
         image: {{ .Values.imageRegistry.registry }}/{{ .Values.imageRegistry.project }}/cyclone-server:{{ .Chart.AppVersion }}
         imagePullPolicy: IfNotPresent
+        args:
+        - "--namespace"
+        - {{ .Release.Namespace }}
```

Comment:
What's the purpose of `namespace` to server ?

Reply:
The server will get cyclone server configmap from this namespace.